event_routing_backends/helpers.py

```diff
@@ -242,8 +243,16 @@ def get_block_id_from_event_data(data, course_id):
     if data is not None and course_id is not None:
         data_array = data.split('_')
         course_id_array = course_id.split(':')
-        block_id = "block-v1:{}+type@problem+block@{}".format(course_id_array[1], data_array[1]) \
-            if len(data_array) > 1 and len(course_id_array) > 1 else None
+        block_version = get_block_version(course_id)
+        if len(data_array) > 1 and len(course_id_array) > 1:
+            block_id = BLOCK_ID_FORMAT.format(
+                block_version=block_version,
+                course_id=course_id_array[1],
+                block_type='problem',
+                block_id=data_array[1]
+            )
+        else:
+            block_id = None # pragma: no cover
```

Comment:
This needs 2 spaces after the `None`, sorry:

`event_routing_backends/helpers.py:255:28: E261 at least two spaces before inline comment`